e2e/cypress/tests/integration/ad_ldap/saml_ldap_sync_remove_spec.ts

```diff
@@ -69,10 +69,10 @@ describe('AD / LDAP', () => {
 
         // # Run LDAP Sync
         const admin = getAdminAccount();
-        cy.runLdapSync(admin);
+        cy.runLdapSync(admin as any);
 
         // # REgister as LDAP user
-        cy.createLDAPUser({user: nonLDAPUser});
+        (cy as any).createLDAPUser({user: nonLDAPUser});
```

Comment:
this is not an issue once you update to latest master

Reply:
updated